tools/archetype/binding/src/main/resources/archetype-resources/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -14,19 +20,23 @@
 		</channels>
 
 		<config-description>
-			<parameter name="config1" type="text" required="true">
-				<label>Sample Parameter</label>
-				<description>This is a sample text configuration parameter</description>
+			<parameter name="hostname" type="text" required="true">
```

Comment:
This is a nice improvement. We should change the example code accordingly. e.g.

https://github.com/openhab/openhab-core/blob/949313744d24b888a90a4e904fca67ae32978b94/tools/archetype/binding/src/main/resources/archetype-resources/src/main/java/internal/__bindingIdCamelCase__Configuration.java#L28

Reply:
Good point. I also adapted the i18n.